keps/sig-node/1967-size-memory-backed-volumes/README.md

```diff
@@ -141,115 +174,139 @@ potentially inaccurate volume size based on node configuration.
 Prior behavior is node dependent, and so pod authors had no mechanism
 to control this behavior properly.
 
-* **Can the feature be disabled once it has been enabled (i.e. can we roll back
-  the enablement)?** Yes
+#### Can the feature be disabled once it has been enabled (i.e. can we roll back the enablement?
+
+Yes
+
+#### What happens if we reenable the feature if it was previously rolled back?
 
-* **What happens if we reenable the feature if it was previously rolled back?**
 Pods that run on that node will have memory backed volumes sized based on Linux
 host default.  The sizing may not align with actual available memory for an app.
 
-* **Are there any tests for feature enablement/disablement?**
+#### Are there any tests for feature enablement/disablement?
+
 No, testing behavior with the feature disabled is dependent on node operating
 system configuration.  The point of this KEP is to address that coupling.
 
 ### Rollout, Upgrade and Rollback Planning
 
-* **How can a rollout fail? Can it impact already running workloads?**
+#### How can a rollout fail? Can it impact already running workloads?
+
 If a pod has more allocatable memory than the default node instance behavior
 of taking 50% node instance memory for sizing emptyDir, a pod could potentially
 write more content to the empty dir volume than previously.  This should have
 no impact on rollout of the cluster or workload.  In practice, applications
 that did exhaust the size of the memory backed volume were not portable across
 instance types or would have had to handle running out of room in that volume.
 
-* **What specific metrics should inform a rollback?**
+#### What specific metrics should inform a rollback?
+
 None.
 
-* **Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?**
+#### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
+
 I do not believe this is applicable.
 
-* **Is the rollout accompanied by any deprecations and/or removals of features, APIs, 
-fields of API types, flags, etc.?**
-  Even if applying deprecation policies, they may still surprise some users.
+#### Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?
+
 No.
 
 ### Monitoring Requirements
 
-* **How can an operator determine if the feature is in use by workloads?**
+#### How can an operator determine if the feature is in use by workloads?
+
 An operator can audit for pods whose emptyDir medium is memory and a size limit
 is specified.  It's not clear there is a benefit to track this because it only
 impacts how the kubelet better enforces an existing API.
 
-* **What are the SLIs (Service Level Indicators) an operator can use to determine 
-the health of the service?**
+#### What are the SLIs (Service Level Indicators) an operator can use to determine the health of the service?
+
 This does not seem relevant to this feature.
```

Comment:
How about the general pod startup latencies https://github.com/kubernetes/community/blob/master/sig-scalability/slos/pod_startup_latency.md ?

Reply:
Updated.